doc/project/release_process.rst

```diff
@@ -94,8 +94,15 @@ At that point, the whole process starts over again.
 Release Quality Criteria
 ************************
 
-The current backlog of prioritized bugs shall be used as a quality metric to
-gate the final release. The following counts shall be used:
+Static Analysis results are a fundamental requirement for the
```

Comment:
I would not place this paragraph first, but after the current one and the bug count table, maybe as a note.

The first sentence "Static Analysis results are a fundamental requirement for the project." seems a bit off topic in the "Release Quality Criteria" section, and more in place in the "Static Code Analysis" section.